consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/validation/MessageValidator.java

```diff
@@ -97,9 +97,13 @@ private boolean validateSignedProposalPayload(final SignedData<ProposalPayload>
     }
 
     final Block proposedBlock = msg.getPayload().getBlock();
-    if (!headerValidator.validateHeader(
-        proposedBlock.getHeader(), parentHeader, protocolContext, FULL)) {
-      LOG.info("Invalid Proposal message, block did not pass header validation.");
+
+    final Optional<BlockValidationOutputs> validationResult =
```

Comment:
I feel like `validationResult` is a misleading name for this - I'd expect it to either be empty to indicate valid or have an enum to give the reason it was invalid.  This is maybe more `blockProcessingOutputs`?  The `BlockValidationOutputs` class might be better as `BlockProcessingOutputs` as well. Not sure there's an ideal name in any case.